lib/nde-repository.test.ts

```diff
@@ -3,13 +3,30 @@ import { NdeRepository } from './nde-repository'
 
 describe('Query SPARQL', async () => {
```

Comment:
Is het mogelijk de tests weer in te schakelen?

Reply:
ik zal dit proberen op te pakken in een andere PR.